applications/asset_tracker_v2/tests/json_common/src/main.c

```diff
@@ -217,8 +217,9 @@ static void test_encode_environmental_data_object(void)
 	 * encoding.
 	 */
 	struct cloud_data_sensors data = {
-		.hum = 50,
-		.temp = 23,
+		.humidity = 50,
+		.temperature = 23,
+		.pressure = 101,
```

Comment:
It's defined as `double` so, either change the type to integer, or use double in the test data.

Reply:
This test validates the structure of the resulting JSON object based on the validation strings in `json_validate.h`. We cannot use decimal values here because the JSON encoder will produce decimal values with slight variations each time. So it cannot be checked against a known value.

There is a separate test that validates the environmental variables using decimal points, `test_floating_point_encoding_environmental`.

Reply:
Hm, ok.